FreeRTOS/FreeRTOS-Plus-TCP
# IPv6/multi: Update the TTL field of ICMP return packet

Description
-----------
According to RFC 1122 section 3.2.1.7 - any device should not send any packet with TTL 0. FreeRTOS+TCP updates the TTL of TCP and UDP packets. This PR adds such ability to the ICMP packet processing as well.

Test Steps
-----------
Send a ping with TTL 0 to the device. FreeRTOS+TCP will return the packet as is (i.e. with packet TTL unchanged - set to 0)

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
